### DISCUSSION TITLE
Custom domain with aliases and full accounts?

### DISCUSSION THREAD

**Discussion by valZho** [simple-login/app#2206] [Feature Requests]
I don't know if this is even possible at all, but had this thought.

Right now for a custom domain, SimpleLogin takes over DNS with it's own primary (mx1.simplelogin.co - priority 10) and secondary (mx2.simplelogin.co - priority 20) mail servers. Thus, if I set up, for example, MyCustomDomain.com, I would not be able to have a full email account ("mailbox") at that domain as that domain is now relegated to forwarding only.

Would it work, would it even be possible, to have a third mx entry pointing to the custom domain; something like:

- domain: @ -- priority: 10 -- target: mx1.simplelogin.co.
- domain: @ -- priority: 20 -- target: mx2.simplelogin.co.
- domain: @ -- priority: 30 -- target: MyCustomDomain.com.

Then on the SimpleLogin side, if there's an unknown (un-aliased) address coming in on that domain (and I would have to imagine that you would have to specifically enable this feature per custom domain), instead of returning a 550 user-not-found error it "fails" over to the custom domain host for handling there.

In theory this would allow any email addresses not specifically handled by SimpleLogin to be handled by the user's host, and thus, you could have a single mailbox and aliases pointing to it all on one domain. As I write this, I can imagine that there are likely problems that would arise from this kind of approach — still, I would love to see some way that I can have both SimpleLogin aliases and full email accounts on the same domain.

**Comment by fm** [comment]
No, because not all servers will retry in order of priority - so you can't reliably rely on that. The only current supported approach is the. use of subdomains.
